tensorboard/plugins/mesh/BUILD

```diff
@@ -163,7 +194,25 @@ py_binary(
     deps = [
         ":demo_utils",
         ":summary",
-        "//tensorboard/compat:tensorflow",
+        "//tensorboard:expect_absl_installed",
```

Comment:
I think this needs to be `//tensorboard:expect_absl_app_installed` and `//tensorboard:expect_absl_flags_installed` (also for the v2 demo).  We have separate rules since internally they are separate dependencies.